santoku/salesforce/lightning.py

```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
             records.extend(response_dict["records"])
         return records
 
+    def soql_response_to_dataframe(
+        self,
+        response: Dict[str, Any],
+        column_mapping: Dict[str, str] = None,
+        drop_columns_containing: str = None,
+    ) -> pd.DataFrame:
+        """
+        Flattens a SOQL response and converts it into a pandas DataFrame.
+
+        To flatten the response json we use the function json_normalize from Pandas [1]

```

Comment:
The response **JSON** object is flattened using the Pandas json_normalize function [1]